File: openapi/code_samples/go/api_v1_organizations_{id}/get.go
scalekitClient := scalekit.NewScalekitClient(
  <SCALEKIT_ENVIRONMENT_URL>,
  <SCALEKIT_CLIENT_ID>,
  <SCALEKIT_CLIENT_SECRET>
)

organization, err := scalekitClient.Organization.GetOrganization(
  ctx,
  organizationId
)

File: openapi/code_samples/javascript/api_v1_organizations_{id}/get.js
import { ScalekitClient } from '@scalekit-sdk/node'
const scalekit = new ScalekitClient(
  process.env.SCALEKIT_ENV_URL,
  process.env.SCALEKIT_CLIENT_ID,
  process.env.SCALEKIT_CLIENT_SECRET,
)
const organization = await scalekit.organization.getOrganization(organization_id)
